File: javascript/1662-arrayStringsAreEqual.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
 var arrayStringsAreEqual = function(word1, word2) {
  let str1 = "";
  for(let i = 0; i < word1.length; i++) {
      str1 += word1[i];
  }

  console.log(str1);

  let str2 = "";
  for(let i = 0; i < word2.length; i++) {
      str2 += word2[i];
  }

  console.log(str2);
  let result = false;
  result = str1 == str2 ? true : false;
  return result;
};

File: java/1662-arrayStringsAreEqual.java
class Solution {
  public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
      String str1 = "";
      for(int i=0; i < word1.length; i++) {
          str1 += word1[i];
      }

      System.out.println(str1);

      String str2 = "";
      for(int i=0; i < word2.length; i++) {
          str2 += word2[i];
      }

      System.out.println(str2);

      boolean result = false;
      if(str1.equals(str2)) {
          result = true;
      }

      return result;
  }
}